<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CustomerLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="DateLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CustomerLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="ItemsTable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addItemButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeItemButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveInoviceButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Invoice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadLastInvoice">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Invoice</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSnack_Shopes_Invoice">
    <property name="title">
     <string>Snack Shopes Invoice</string>
    </property>
   </widget>
   <addaction name="menuSnack_Shopes_Invoice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
